README.md

```diff
@@ -64,6 +65,9 @@ using `helm template`.
       poetry run sh -c '${VIRTUAL_ENV}/lib/python3.*/site-packages/kapitan/dependency_manager/helm/build.sh'
       ```
 
+    Install jsonnet-bundler according to upstream [documentation](https://github.com/jsonnet-bundler/jsonnet-bundler#install).
+    For OS X, jsonnet-bundler is available as [Homebrew](https://brew.sh/) package.
```

Comment:
I'd prefer to leave this line out, otherwise it gives the feeling that it's OS X biased or we would need to add the info for other OS'es as well. I'd leave how to install it up to the user.